core/src/test/java/io/questdb/test/griffin/EngineMigrationTest.java

```diff
@@ -527,26 +527,26 @@ private void assertColTopsO3() throws SqlException {
         );
 
```

Comment:
Previous result was wrong because factory skipped raw rows assuming no filtering  and default data access.